packages/astro/src/compiler/codegen/index.ts

```diff
@@ -225,7 +224,7 @@ function getComponentWrapper(_name: string, hydration: HydrationAttributes, { ur
       }
     };
 
-    let metadata: string = '';
+    let metadata = '';
```

Comment:
why remove this type? is this variable not a string?

Reply:
The value is a string so the type isn't necessary